### PR TITLE
Add GUIDE.md, CONTRIBUTING.md and CODE_OF_CONDUCT.md

### LINKED ISSUE #28: Create GUIDE.md, CONTRIBUTING.md and CODE_OF_CONDUCT.md
Document everything in an organized way for making it easy for contributors and users to get started with the project.

### PR DESCRIPTION
**Pull request by ruddi10**
Add proper documentation to help new users to get started with the project. fixes #28

### CODE REVIEW

**Comment by ruddi10** [comment]
@muskankhedia PTAL

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+## Code of Conduct
+
+This project and everyone participates in it is governed by this [Code of Conduct](CODE_OF_CONDUCT.md).
+By participating, you are expected to uphold this code. Please report unacceptable behavior to [**benchroutes@gmail.com**](mailto:benchroutes@gmail.com).
+
+## How Can I Contribute?
+
+Firstly, if you are new to the project, kindly go through [README.md](https://github.com/bench-routes/dashboard/blob/master/README.md) to setup the project
+in your local machine. Then have a quick look at [GUIDE.md]() to understand working of all the components present in the project. You can also have a look at our project [design documentation]() to have more insights on our project structure and design.
```
Comment (muskankhedia) [inline CONTRIBUTING.md:16]: Please provide the link here for GUIDE.md and design documentation

Comment (ruddi10) [reply CONTRIBUTING.md:16]: Ohh missed that thanks

**Comment by ruddi10** [comment]
@muskankhedia  DONE

**Review by muskankhedia — APPROVED** [review]
Looks good!

**Pull request merged**